service/arm_templates/security_groups_setup.json

```diff
@@ -198,7 +172,7 @@
               "protocol": "tcp",
               "sourcePortRange": "*",
               "destinationPortRange": "[string(parameters('kubernetesIngressSecurePort'))]",
-              "sourceAddressPrefix": "[parameters('ingressLoadBalancerCidr')]",
+              "sourceAddressPrefix": "*",
```

Comment:
Nitpick: I think the description should be updated.

Reply:
Description updated